From dcd61438879460ab0d7a412c175cc5fcbab26ee4 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Tue, 23 Apr 2019 13:23:19 +0200
Subject: [PATCH] tqma8mx: implement board_mmc_env_dev and
 mmc_map_to_kernel_blk

missing from conversion to new u-boot. leads to e-mmc boot error

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 board/tqc/tqma8mx/tqma8mx-mba8mx.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/board/tqc/tqma8mx/tqma8mx-mba8mx.c b/board/tqc/tqma8mx/tqma8mx-mba8mx.c
index 484df33f34..85f98910b0 100644
--- a/board/tqc/tqma8mx/tqma8mx-mba8mx.c
+++ b/board/tqc/tqma8mx/tqma8mx-mba8mx.c
@@ -194,6 +194,20 @@ int tqc_bb_board_init(void)
 	return 0;
 }
 
+/*
+ * SD1 -> mmc0 / mmcblk0
+ * SD2 -> mmc1 / mmcblk1
+ */
+int board_mmc_get_env_dev(int devno)
+{
+	return devno;
+}
+
+int mmc_map_to_kernel_blk(int dev_no)
+{
+	return dev_no;
+}
+
 int tqc_bb_board_late_init(void)
 {
 #ifdef CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
